regression/regression.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import path from 'path';
+import { Calculator } from '../src';
+
+const regressionBaseName = process.argv[2] ?? 'regression-output';
+const verbose = process.argv[3] === 'true';
+
+const REGRESSION_OUTPUT_DIR = path.join(__dirname, `./output/${regressionBaseName}`);
+const CTHON_BASE_PATH = path.join(__dirname, './connectathon/fhir401/bundles/measure');
+const ECQM_CONTENT_BASE_PATH = path.join(__dirname, './ecqm-content-r4-2021/bundles/measure');
+const ECQM_CONTENT_QICORE_BASE_PATH = path.join(__dirname, './ecqm-content-qicore-2022/bundles/measure');
```

Comment:
Just out of curiosity, why are we including this? It looks like there is only one measure bundle and no test patients so there is no output.

Reply:
Mainly just future proofing. I didn't notice that there weren't any test patients for those, but hopefully there are eventually.